### ISSUE TITLE
Default settings

### ISSUE THREAD

**Issue by ardove** [RicoSuter/NSwag#2993]
Is there a way to include default settings that are always configured when generating client code? It'd be exceptionally useful not to need to select the 10 options I want to re-configure every time I use NSwagStudio.

**Comment by RicoSuter** [comment]
You can save the configuration as a file in NSwagStudio so that you can regenerate clients quickly or via CLI.

**Comment by ardove** [comment]
Yeah, but I can only save and load configuration for a specific client that I've already configured. I want configuration that is loaded by default when I want to generate a new client (otherwise I have to re-configure all of those settings).

**Comment by RicoSuter** [comment]
The config is a JSON file, why dont you create a template and copy this?

**Comment by ardove** [comment]
Right, I understand that. We could certainly create a dotnet template or something which produces the correct configuration given a url, but, at this point, we plan on using NSwagStudio to manage references to downstream services in the form of an nswag document just like we do with wcf services. The primary reason is that it allows developers to configure a service reference directly in visual studio which is an operation that they're already familiar with. 

I'd like to have the same sort of functionality with the NSwag client so that it's not a big mental leap for someone to figure out how to configure multiple service references quickly (in this case, multiple http api endpoints)

It'd be nice if I could just configure the options I want once and then generate nswag files for different swagger uris without having to tweak the config settings each time.
